<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-FBE7IIT" failures="1" tests="7" name="Test" time="0.42" errors="0" timestamp="2023-11-24T16:42:31 IST">
  <testcase classname="practise.RerunFailedTestCase" name="logIn" time="0.025"/>
  <testcase classname="practise.RerunFailedTestCase" name="homePage" time="0.006"/>
  <testcase classname="practise.RerunFailedTestCase" name="addToCart" time="0.064">
    <skipped/>
  </testcase> <!-- addToCart -->
  <testcase classname="practise.RerunFailedTestCase" name="addToCart" time="0.011">
    <skipped/>
  </testcase> <!-- addToCart -->
  <testcase classname="practise.RerunFailedTestCase" name="addToCart" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at practise.RerunFailedTestCase.addToCart(RerunFailedTestCase.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- addToCart -->
  <testcase classname="practise.RerunFailedTestCase" name="transaction" time="0.02"/>
  <testcase classname="practise.RerunFailedTestCase" name="logOut" time="0.013"/>
</testsuite> <!-- Test -->
